apps/webapp/src/app/login/login.tsx

```diff
@@ -3,11 +3,13 @@ import { Button, TextField } from '@mui/material';
 import styled from '@emotion/styled';
 import { useState } from 'react';
 import { useAuthentication } from '../auth/use-authentication';
+import { useDocumentTitle } from '../document-title/use-document-title';
 import { v4 } from 'uuid';
 
 export const Login = () => {
   const [name, setName] = useState<string>();
   const { dispatch: authDispatch } = useAuthentication();
+  useDocumentTitle('Witaj ' + (name ?? ''));
```

Comment:
po prostu cebulendarz? w momencie jak będziesz znał name to właśnie będziesz zamykał ekran wla w trakcie pisania to mało wnosi. chciałbyś robić zakładkę o tytule "Witaj Dominik"? ;)

imo, tu i na głównej ponprostu nazwa apki.

a będziesz robił czyszczenie efektem? bo jak tak to nie musimy nic robić w tych widokach

Reply:
widzę że zrobiłeś czyszczenie, to już nie jest
potrzebne dawać tytuły w ekranach ktróre nie mają sensownego kontekstu

Reply:
Dobra, czuję się solidnie przeczołgany na tej mikrozmianie :D 
Pora na prawdziwą batalię (coś sobie znajdę) xD

Reply:
😁